src/ng/directive/ngTransclude.js

```diff
@@ -159,35 +159,45 @@
  * </example>
  */
 var ngTranscludeMinErr = minErr('ngTransclude');
-var ngTranscludeDirective = ngDirective({
-  restrict: 'EAC',
-  link: function($scope, $element, $attrs, controller, $transclude) {
+var ngTranscludeDirective = ['$compile', function($compile) {
+  return {
+    restrict: 'EAC',
+    terminal: true,
```

Comment:
There was a problem in this PR with failing to link fallback content if there was a no content provided for an optional named slot. My #14787 fixes that